tools/add_triage_signature.py

```diff
@@ -713,6 +713,49 @@ def _process_ticket(self, url, issue_key):
         report = self._generate_table_for_report(hosts)
         self._update_triaging_ticket(report)
 
+class MasterFailedToRebootSignature(ErrorSignature):
+    """
+    This signature finds tickets for clusters where at least one master node failed to reboot
+    This signature is only applicable if the host that failed to reboot has bios boot mode
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            comment_identifying_string="h1. Master node failed to reboot",
+            function_impact_label="master_failed_rebooting",
+        )
+
+    def _process_ticket(self, url, issue_key):
+        md = get_metadata_json(url)
+        cluster = md["cluster"]
+        hosts = []
+        for host in cluster["hosts"]:
+            inventory = json.loads(host["inventory"])
+            boot_mode = inventory.get("boot", {}).get("current_boot_mode", "N/A")
+            if (
+                host["role"] == "master"
```

Comment:
Does this exclude bootstrap? Do we want to exclude bootstrap?

Reply:
It doesn't, in case a master node (with BIOS boot mode) failed to reboot we will label it

Reply:
Updating the logic here